styleguide/source/_patterns/02-molecules/article-stub.twig

```diff
@@ -20,31 +20,30 @@
   {% set related = articleStub.related %}
   {% set link = articleStub.link %}
   {% set paragraph = articleStub.paragraph %}
-  {% set readtime = articleStub.readtime %}
-  {% set date = articleStub.date %}
-
 
   {% if paragraph and not title and not related %}
     <div class="ama__article-stub__description">
       {% include "@atoms/paragraph.twig" %}
     </div>
   {% endif %}
-  {% if readtime and date and not related %}
-    <div class="ama__metadata article-stub__metadata col-xs-12">
-      {% include "@molecules/article-metadata.twig" %}
-    </div>
+{% endmacro %}
+
+{% macro metadata(articleStub) %}
+  {% set metadata = articleStub.metadata %}
+  {% if metadata and not title and not related %}
+    {% include "@molecules/article-metadata.twig" %}
```

Comment:
The metadata appears to be coming in on the `article-stub-related` and I don't believe that it should be there based on what I've seen in the News Article wireframe and the current Topics page.

<img width="778" alt="screen shot 2018-01-24 at 4 13 59 pm" src="https://user-images.githubusercontent.com/1653723/35359994-cab06f7c-0121-11e8-9824-2a8c3073d47f.png">

Reply:
Metadata is also appearing in the `article-stub-title-only` when it should not be there. 

<img width="370" alt="screen shot 2018-01-24 at 4 20 09 pm" src="https://user-images.githubusercontent.com/1653723/35360261-c5c070c4-0122-11e8-8846-fe6f4a3945b2.png">

Reply:
yeah. that was an oversight. will fix